indexing/src/main.cpp

```diff
@@ -847,8 +847,7 @@ namespace
 			}
 #if 0
 			// catalog / index drift correction
-			if(irods::indexing::schedule::indexing ==
-			   rule_obj["rule-engine-operation"]) {
+			if(irods::indexing::schedule::indexing == re_operation) {
```

Comment:
`re_operation` isn't declared for this.

Fixing it now or opening an issue to address it later is fine.